packages/components/src/Tabs/TabPanels.tsx

```diff
@@ -62,7 +62,14 @@ const Layout: FC<TabPanelsProps> = ({
   )
 
   return (
-    <div className={className} {...tabPanelsLayoutProps}>
+    <div
+      aria-labelledby={`tab-${selectedIndex}`}
+      className={className}
+      id={`panel-${selectedIndex}`}
+      role="tabpanel"
+      tab-index={selectedIndex}
```

Comment:
`tabIndex` here as well :)

```suggestion
      tabIndex={selectedIndex}
```